src/app/services/lunch.service.ts

```diff
@@ -22,7 +22,7 @@ export class LunchService {
 				if (data.error) {
 					throw new Error(data.error);
 				}
-
+				console.log(data.lunch);
```

Comment:
Try to remove all debugging `console.log`s before committing, or if you're going to use the `console.log` for a while, at least remove it before merging.